corehq/apps/accounting/migrations/0043_add_web_user_to_payment_methods.py

```diff
@@ -0,0 +1,317 @@
+# -*- coding: utf-8 -*-
+from south.db import db
+from south.v2 import DataMigration
+from django.db import connection
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        sql = """
+            UPDATE accounting_paymentmethod
```

Comment:
In rails I used to run into issues when copying data from models which were marked for deletion. If someone came along at a later date and were to run this migration, since `BillingAccountAdmin` no longer existed in the code base, the migration would fail. I didn't look into this in Django, but the raw SQL was basically habit / me being lazy in looking up the django specifics here.
What happens in migrations if the python models no longer exist in the codebase?